src/services/users/reset-password/verifyPasswordResetToken.js

```diff
@@ -0,0 +1,18 @@
+const jwt = require('jsonwebtoken')
```

Comment:
for now, let's go with the hyphenated file name style for a bit of consistency. `verify-password-reset-token.js`